README.md

```diff
@@ -1,24 +1,201 @@
+
+- []()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎬 Kanban Board <a id="kanban"></a>
+
+Link to the [Kanban Board](https://github.com/users/binodbhusal/projects/4/views/1)
+
+All 4 Authors have been contributing to this project from start to finish
+
+You can find the initial snapshot and setup of the Kanban board [here](https://github.com/binodbhusal/final_capstone_backend/issues).
+
+## Video Presentation <a name="video-presentation"></a>
+
+Click on this [Coming Soon](#) to view the video walkthrough of this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Installed Git.
+- Installed Ruby
+- Installed Ruby on Rails
+- Create a local directory that you want to clone the repository.
+- Open the command prompt in the created directory.
+- On the terminal run this command git clone https://github.com/binodbhusal/final_capstone_backend.git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone https://github.com/binodbhusal/final_capstone_backend.git
+```
+
+```sh
+cd final_capstone_backend
+```
+
+```sh
+bundle install
+```
+
+```sh
+bundle exec rails db:create db:migrate
+```
+
+```sh
+rails s
+```
+
+### Run Tests
+
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+### 👤 **Dieum Akonkwa**:
+
+- GitHub: [@dieum512](https://github.com/dieum512)
+- Twitter: [@twitter](https://twitter.com/dieumakonkw)
+- LinkedIn: [@linkedin]()
+
+### 👤 **Alijan Rahimi**
+
+- GitHub: [@rahimialijan](https://github.com/rahimialijan)
+- Twitter: [@twitter](https://twitter.com/AlijanRahimi10)
+- LinkedIn: [@linkedin](https://www.linkedin.com/in/alijan-rahimi-18389ab3/)
+
+### 👤 **Binod Bhusal**
+
+- GitHub: [@binodbhusal](https://github.com/binodbhusal)
+- Twitter: [@twitter](https://twitter.com/Binod_ironLad)
+- LinkedIn: [@linkedin](https://www.linkedin.com/in/binodbhusal/)
+
+### 👤 **Salomon Tshobohwa**:
+
+- GitHub: [@Tshobohwa ](https://github.com/Tshobohwa)
+- LinkedIn: [@linkedin](https://www.linkedin.com/in/tshobohwasalomon/)
+
+# 🔭 Future Features:
+
+- Proper authentication
+- Authorization
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you liked this project, give it a ⭐️ and kindly send to me an e-mail expressing it, it would make our day and fuel our motivation.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for this opportunity.
+
```

Comment:
update here like this:
Hats off to [Murat Korkmaz](https://www.behance.net/muratk) for designing the awesome template that was used. ✨
Microverse for giving us this chance
The amazing code reviewers for making us improve every day 👍